homeassistant/components/plant/__init__.py

```diff
@@ -183,6 +185,12 @@ def __init__(self, name, config):
             self._conf_check_days = self._config[CONF_CHECK_DAYS]
         self._brightness_history = DailyHistory(self._conf_check_days)
 
+        for reading in self.READINGS.values():
+            if "min" in reading and reading["min"] in self._config:
+                self._limitsmap[reading["min"]] = self._config[reading["min"]]
+            if "max" in reading and reading["max"] in self._config:
+                self._limitsmap[reading["max"]] = self._config[reading["max"]]
```

Comment:
You could even cut down one line by using a for loop with a list:
```suggestion
            for item in ["min", "max"]:
                if item in reading and reading[item] in self._config:
                    self._limitsmap[reading[item]] = self._config[reading[item]]
```